tests/integration/openebs/customdatadir_test.go

```diff
@@ -0,0 +1,53 @@
+package openebs
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/addons/openebs"
+	"github.com/replicatedhq/embedded-cluster/pkg/defaults"
+	"github.com/replicatedhq/embedded-cluster/tests/integration/util"
+	"github.com/replicatedhq/embedded-cluster/tests/integration/util/kind"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOpenEBS_CustomDataDir(t *testing.T) {
+	t.Parallel()
+
+	clusterName := util.GenerateClusterName(t)
+	kindConfig := util.NewKindClusterConfig(t, clusterName, nil)
+
+	dataDir := util.TmpNameForHostMount(t, "data-dir")
```

Comment:
why not just use `t.TempDir`?

Reply:
i thought there was an issue mounting t.TempDir() but it turns out it was i forgot to defer

Reply:
this didnt work